test/spanner.ts

```diff
@@ -225,12 +225,13 @@ describe('Spanner with mock server', () => {
         SimulatedExecutionTime.ofError(err)
       );
       database.run(selectSql, (err, rows) => {
-        if (err) {
-          assert.fail(err);
-        } else {
-          assert.strictEqual(rows!.length, 3);
-        }
-        done();
+        assert.ifError(err);
+        assert.strictEqual(rows!.length, 3);
+        database
+          .close()
+          // This will cause done to be called with an error and fail the test.
```

Comment:
Since you're not adding the same comment in all the other places with the same sequence, I would remove this comment.

Reply:
Done.